src/main/resources/db/migration/V11__synchronization_table.sql

```diff
@@ -0,0 +1,10 @@
+create table if not exists synchronization
```

Comment:
Hvordan funker det egentlig med flyway når vi har scriptene V10, V12, V13 og V14, og så kommer V11..? Vil det feile? Litt rart at man har hoppet over 11 forøvrig 😅

Reply:
Trenger vi denne egentlig når alle tabellene uansett har kolonner for sist oppdatert?

Reply:
Snodig, jeg committa egentlig scriptet for lenge siden og det er derfor det er versjon 11.
Jeg må rename scriptet ja, men skjønner da ikke hvorfor det ikke finnes en annen med versjon 11

Reply:
Bestemte at det skulle være en annen tabell i diskusjon med Tormod og Magnus. Grunnen er at tabellen oppdateres med data fra forskjellige kilder. 
For eksempel så kan man få ny nav veileder uten at det henger sammen med om kontaktinfo har endret seg. 
De ønsket også en tabell som skulle ha støtte for å vise synkronisering av andre data på andre tabeller

Reply:
Jeg tror kanskje jeg hoppa over V11 for denne PRen var egentlig klar før min 😅 også ble det sikkert glemt å endre.

Tanken med tabellen, var egentlig å kunne skille mellom hva som er oppdatert, hvis navnet til en bruker er nylig oppdatert betyr ikke det at vi ikke vil sjekke om de har fått nytt telefonnummer.

Reply:
Okei :) Stusset litt siden det bare er KRR den brukes for nå. Det at den brukes for andre datatyper kommer kanskje i en egen pr?

Reply:
jeg har ingen umiddelbare planer om å gjøre det. 
Første alternativ var å lage en egen kolonne i nav_bruker tabellen og kalle den "krr_last_sync" eller noe sånt.
Spiller ikke så stor rolle for meg om man gjør det på den ene eller andre måten men for en tjeneste som har som ansvar å synce data fra mange kilder så gir det mening at man separerer det ut

Reply:
Hvis det ikke er noen planer om at det skal brukes for de andre datatypene så synes jeg det kanskje blir litt vel komplisert å gjøre det sånn. Det er jo noe med å løse de behovene vi faktisk har her og nå. Men når det allerede er gjort så kan det sikkert være sånn 🤷‍♀️

Reply:
Har ikke noe mot å gjøre det om, om det skulle være mest hensiktsmessig. Hva tenker @tofoss  og @magnusev ?

Reply:
Er den "Over engineered" for dette, kanskje. Men jeg er heller ikke noe fan av å ha et felt som heter `sistKrrOpdatert` i navBruker heller 🤷